deployer/deployer-services/src/main/java/io/hyscale/deployer/services/factory/PodParentFactory.java

```diff
@@ -42,5 +45,16 @@ public static void registerHandlers() {
     public static PodParentHandler getHandler(String kind) {
         return kindVsHandlerMap.get(kind);
     }
+    
+    /**
+     * 
+     * @return Unmodifiable list of all available PodParentHandlers
+     */
+    public static List<PodParentHandler> getHandlersList(){
```

Comment:
getAllHandlers ? What do you say ?

Reply:
Used the nomenclature we are using in ResourceHandlers.
We can go with getAllHandlers at both the places?

Reply:
Yes